modules/api/src/main/java/org/apache/ignite/table/IgniteTableStatistics.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.table;
+
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Table statistics facade provides methods for managing table statistics.
+ *
+ * <p>Table statistics are used by SQL engine for SQL queries planning.
+ */
+// TODO: drop this class and move all the methods to Table interface?
+public interface IgniteTableStatistics {
+    /**
+     * Returns all the statistics names for the table.
+     *
+     * @return Statistics names for the table.
+     */
+    Collection<String> statisticNames();
+    
+    /**
+     * Get statistics info for the table.
+     *
+     * @param statisticName Statistic name.
+     * @return Statistics info.
+     */
+    StatisticInfo statistic(String statisticName); //TODO: Local or global? Ready or in-progress? TBD.
```

Comment:
There is nothing about "local or global" and "ready or in-progress" terms in javadoc. What do these terms mean?